tests/ShadowDOM/js/Document.js

```diff
@@ -230,14 +233,18 @@ htmlSuite('Document', function() {
 
     div.offsetHeight;
 
-    var list = document.querySelectorAll('div /deep/ aa');
```

Comment:
is there a reason why some tests look duplicated in Document.js & Element.js (see below)